### PR TITLE
Make implicit TRUE permissions check explicit in API 4 Explorer for PHP

### PR DESCRIPTION
**Pull request by larssandergreen**
Overview
----------------------------------------
It feels like we end up with quite a few bugs because of the implicit TRUE permissions checks in API 4. I think it might make life a little easier if we make the implicit TRUE explicit in the API 4 Explorer, so if we're copying and pasting we're more likely to stop and think if we should be checking permissions or not.

This only changes the PHP.

The downside is possibly a few more lines of code in core when using traditional style PHP, but that seems well worth it if we avoid a few bugs.

### CODE REVIEW

File: ang/api4Explorer/Explorer.js [hunk 1]
```diff
@@ -876,9 +878,8 @@
         newLine = "\n" + _.repeat(' ', indent),
         code = '\\' + info.class + '::' + action + '(',
         args = _.cloneDeep(info.class_args || []);
-      if (params.checkPermissions === false) {
-        args.push(false);
-      }
+        // Always shows implicit true permissions check for PHP
+        args.push((params.checkPermissions === false) ? false : true);
```
Comment (colemanw) [inline ang/api4Explorer/Explorer.js:882]: This indentation is wrong, needs to be un-indented by 2 spaces.

**Comment by larssandergreen** [comment]
@colemanw Yes, that would have been more readable.

File: ang/api4Explorer/Explorer.js (hunk 1 above)
Comment (larssandergreen) [reply ang/api4Explorer/Explorer.js]: Fixed

**Comment by colemanw** [comment]
@larssandergreen it's *kinda* not too late to add those 2 functions as wrappers to `execute()`... but it would kinda be a breaking change because any code using those new functions would be incompatible with earlier versions of civi where they don't exist.

**Comment by larssandergreen** [comment]
@colemanw We could just use `setCheckPermissions(TRUE|FALSE)` in the API Explorer instead, definitely more readable, though it's an extra line. What if we showed it as `->setCheckPermissions(TRUE|FALSE)->execute();` (which is kind of the same as  you're suggesting).

**Comment by colemanw** [comment]
Ugh, the whole reason I added the shorter syntax was because people complained about the wordiness of `->setCheckPermissions(TRUE|FALSE)->execute();`.
Can't win 🤷‍♂️

**Comment by larssandergreen** [comment]
But we can also merge this as is, that's fine with me too.

**Comment by colemanw** [comment]
Fine with me too :)

**Pull request merged**